### PR TITLE
Update build instructions in README

### PR DESCRIPTION
**Pull request by taste1981**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 # Open WebRTC Toolkit Native SDK
 
 ## Introduction
-The Open WebRTC Toolkit client SDK for native Windows/Linux/Android/iOS applications is built upon the W3C WebRTC standard to accelerate the development of real time communication applications on these platforms. It supports peer to peer and conference mode communication working with Open Media Stream MCU server.
+The Open WebRTC Toolkit(OWT) client SDK for native Windows/Linux/Android/iOS applications is built upon the W3C WebRTC standard to accelerate the development
+of real time communication applications on these platforms. It supports peer to peer communication, and conference mode communication working with
+Open WebRTC Toolkit MCU server.
```
Comment (jianjunz) [inline README.md:6]: MCU server -> conference server or media server.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 
-- Supported Windows platform: Windows 7 and above.
-- Supported Linux platform: Ubuntu 16.04.
+- Supported Windows platform: Windows 8 and above.
+- Supported Linux platform: Ubuntu 16.04 & 18.04.
```
Comment (jianjunz) [inline README.md:9]: Not sure if 16.04 is supported, but 20.04 should be supported as well.

File: README.md [hunk 3]
```diff
@@ -45,34 +47,39 @@ target_os = []
...
-  The built binary will be under `output_path`, the document for sdk will also be copied to this directory if docs have been generated. 
-If `output_path` is not set, the built binary will be under the `src/out` directory. Note the first time you run this it will take a 
-long time to pull chromium/webrtc dependencies and require network access to Google's code/storage infrastructure. Set `ssl_root` to the 
-directory of your OpenSSL 1.1.0 binary. Use `--gn_gen` to generate args.gn during the first build or when you change the `ssl_root` path.
-If `--msdk_root` is specified to correct Intel MediaSDK path, for example, '/opt/intel/mediasdk', hardware codecs will be built besides the
-software implementations. If `--fake_audio` is specified, the internal audio devices implementation based on alsa or pulseaudio will not be built.
+1. In `src` direcotry, run `gclient sync`. It may take long time to download large amount of data.
+1. Go to the `src/scripts` directory, and run: `python build_linux.py --gn_gen --sdk --tests --ssl_root /path/to/ssl --output_path /path/to/out`.
```
Comment (jianjunz) [inline README.md:61]: One more step is needed on Linux.

Run https://chromium.googlesource.com/chromium/src/+/HEAD/build/install-build-deps.sh to install dependencies.

Comment (taste1981) [reply README.md:61]: This is already covered in Prepare the development environment-> WebRTC stack build dependencies link in the same document.

File: README.md (hunk 2 above)
Comment (taste1981) [reply README.md:9]: Will add this when docker file is created with Ubuntu 20.04 base image.

**Review by jianjunz — APPROVED** [review]

**Pull request merged**